### PR TITLE
New package: ShaulEizikovich.vJoyDeviceDriver version 2.1.9.1

### PR DESCRIPTION
**Pull request by rakleed**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/128210)

### CODE REVIEW

**Comment by stephengillie** [comment]
Hi @rakleed,

Manual Validation ended with:
image

Is there an installer switch to bypass this message and complete the install automatically?

**Comment by rakleed** [comment]
@stephengillie in my case, the `/VERYSILENT` flag helped, but for some reason it does not work for you: https://github.com/microsoft/winget-pkgs/pull/128210/commits/bf0f2a41a9c45ae92d44e543ad1dd7b06d2c73b4

File: manifests/s/ShaulEizikovich/vJoyDeviceDriver/2.1.9.1/ShaulEizikovich.vJoyDeviceDriver.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+PackageIdentifier: ShaulEizikovich.vJoyDeviceDriver
+PackageVersion: 2.1.9.1
+InstallerType: inno
+Installers:
+- Architecture: x64
+  InstallerUrl: https://downloads.sourceforge.net/project/vjoystick/Beta%202.x/2.1.9.1-160719/vJoySetup.exe
+  InstallerSha256: F103CED4E7FF7CCB49C8415A542C56768ED4DA4FEA252B8F4FFDAC343074654A
+  InstallerSwitches:
+    Silent: /VERYSILENT
```
Comment (Trenly) [inline manifests/s/ShaulEizikovich/vJoyDeviceDriver/2.1.9.1/ShaulEizikovich.vJoyDeviceDriver.installer.yaml:12]:
```suggestion
    Silent: /VERYSILENT
    SilentWithProgress: /VERYSILENT
```
Winget runs in the `SilentWithProgress` mode by default, but when an application doesn't support progress, the silent switch should be added here as well

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/s/ShaulEizikovich/vJoyDeviceDriver/2.1.9.1/ShaulEizikovich.vJoyDeviceDriver.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: ShaulEizikovich.vJoyDeviceDriver
+PackageVersion: 2.1.9.1
+PackageLocale: en-US
+Publisher: Shaul Eizikovich
+PackageName: vJoy Device Driver
```
Comment (stephengillie) [inline manifests/s/ShaulEizikovich/vJoyDeviceDriver/2.1.9.1/ShaulEizikovich.vJoyDeviceDriver.locale.en-US.yaml:8]:
```suggestion
PackageName: vJoy Device Driver
PackageUrl: https://www.vjoy.org/download-for-windows
```

**Comment by stephengillie** [comment]
Hi @rakleed,

Unfortunately, the developer's page doesn't have any links to Sourceforge, only to Mega.

**Comment by rakleed** [comment]
@stephengillie why did you decide that https://www.vjoy.org/ is the official website? According to whois https://www.whois.com/whois/vjoy.org, this site was created several months ago. And the repository on GitHub https://github.com/shauleiz/vJoy was created 9 years ago, and it contains a link to SourceForge.

**Comment by stephengillie** [comment]
> @stephengillie why did you decide that https://www.vjoy.org/ is the official website? According to whois https://www.whois.com/whois/vjoy.org, this site was created several months ago. And the repository on GitHub https://github.com/shauleiz/vJoy was created 9 years ago, and it contains a link to SourceForge.

Hi @rakleed,

In lieu of other data, an Internet search suggested a site with the project name as the primary component of the domain. Thanks for supplying this project's URL.

File: manifests/s/ShaulEizikovich/vJoyDeviceDriver/2.1.9.1/ShaulEizikovich.vJoyDeviceDriver.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: ShaulEizikovich.vJoyDeviceDriver
+PackageVersion: 2.1.9.1
+InstallerType: inno
+Installers:
+- Architecture: x64
+  InstallerUrl: https://downloads.sourceforge.net/project/vjoystick/Beta%202.x/2.1.9.1-160719/vJoySetup.exe
```
Comment (stephengillie) [inline manifests/s/ShaulEizikovich/vJoyDeviceDriver/2.1.9.1/ShaulEizikovich.vJoyDeviceDriver.installer.yaml:9]:
```suggestion
  InstallerUrl: https://downloads.sourceforge.net/project/vjoystick/Beta%202.x/2.1.9.1-160719/vJoySetup.exe
```

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  
> 2023-12-04 14:36:18.495 [CORE] DeliveryOptimization error: 0x80d02002, extended error: 0x80072ee2 
> 2023-12-04 14:36:18.526 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\DODownloader.cpp(289)\WindowsPackageManager.dll!00007FFCA2451EFD: (caller: 00007FFCA24527D0) Exception(1) tid(1f68) 80D02002

**Review by stephengillie — APPROVED** [review]

**Pull request merged**